subprojects/docs/src/docs/stylesheets/userGuideHtmlCommon.xsl

```diff
@@ -32,9 +32,9 @@
     <xsl:param name="css.decoration">0</xsl:param>
     <xsl:param name="highlight.source" select="1"/>
 
-    <!-- Use custom style sheet content -->
-    <xsl:param name="html.stylesheet">DUMMY</xsl:param>
-    <xsl:template name="output.html.stylesheets">
+    <!-- Customize the metadata and stylesheets to add to the <head> element -->
+    <xsl:template name="head.content">
```

Comment:
I'm afraid this completely overwrites the current head content, and a bunch of other headers were now omitted. I've reverted this commit.

Reply:
> I'm afraid this completely overwrites the current head content, and a bunch of other headers were now omitted.

@eriwen You're right, thanks for pointing that out. In the user guide, my changes removed the following two lines:

```
<title>Gradle User Guide Version 4.4-20171127220000+0000</title>
<meta content="DocBook XSL Stylesheets V1.75.2" name="generator">
``` 

The same lines are also removed in the DSL reference. I'll try to see if I can keep these two lines along with the `<meta>` tag I added.

## User guide <head> after your revert (i.e. before my changes)

```
git checkout dc30798574d99957cec500a893f00f78890577a7
./gradlew docs:userguide
```

![screenshot](https://user-images.githubusercontent.com/1498164/33344611-38f82dba-d492-11e7-81f2-d1e4b7d93cf5.png)

## User guide <head> before your revert (i.e. after my changes)

```
git checkout dc30798574d99957cec500a893f00f78890577a7^
./gradlew docs:userguide
```

![screenshot](https://user-images.githubusercontent.com/1498164/33344436-ad125aa0-d491-11e7-9b29-632d82fbfb37.png)

Reply:
@eriwen My bad, it seems like you've already fixed the issue: 0a77f4b4b67563c569bd55a518446637e8349cb8. However, this commit only adds the `viewport` meta tag to the user guide and not the DSL reference. But we discussed this privately and decided that we didn't want to add the `viewport` meta tag to the DSL reference, so this change is fine and no more changes are required.